modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -280,6 +281,16 @@ exports.send = function(req, res) {
       });
     },
 
+    // Here we collect some message data into the influxdb
+    function (message, done) {
+      // the module returns a Promise
+      // it is async, but we'll let it run wild (not waiting for the promise to
+      // be fulfilled or rejected)
+      messageToInfluxService(message);
```

Comment:
I've used some libraries where if I fail to pass a function to the `.then()` of the promise, the work never gets done, and the whole thing is silently ignored. If this works, maybe it's fine to leave it. It might also be safer (and potentially clearer) to use something like:

    messageToInfluxService(message).then(function() {}).catch(function() {})

This also makes it clear that we don't care about waiting.

How about this for the comment:

> We don't want to wait for a result if the metrics collection succeeds or fails, we silently ignore any success or error messages.